fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/mixin/content/registry/MixinFireBlock.java

```diff
@@ -53,7 +54,7 @@ private void afterConstruct(Block.Settings settings, CallbackInfo info) {
 
 	@Inject(at = @At("HEAD"), method = "getBurnChance", cancellable = true)
 	private void getFabricBurnChance(BlockState block, CallbackInfoReturnable info) {
-		FlammableBlockRegistry.Entry entry = fabric_registry.getFabric(block.getBlock());
+		FlammableBlockRegistry.@Nullable Entry entry = fabric_registry.getFabric(block.getBlock());
```

Comment:
This is a weird syntax. Can't the annotation be put before the full type declaration ?

Reply:
That means the annotation is applied to `FlammableBlockRegistry` instead of `Entry`, which causes compile error.

Reply:
I find it a bit ugly but ok

Reply:
You are yet to see truly ugly ones like `@Nullable Object[]` (the element type is nullable) vs `Object @Nullable[]` (the array type is nullable)